aries-site/src/data/structures/templates.js

```diff
@@ -251,4 +251,23 @@ export const templates = [
     sections: [],
     relatedContent: ['Table', 'Lists', 'Filtering'],
   },
+  {
+    name: 'Internationalization',
+    available: false,
```

Comment:
```suggestion
    available: true,
```
Should this be true?

Reply:
Good question. I wasn't quite sure why a couple spots sort by this. Thoughts?

Reply:
`available` property is used in to indicate we have Design System styling and guidance. When sorting, it is used to more prominently feature components/templates cards for which we do have styling and guidance.

This designation was relevant in the early days of the Design System, but as the coverage across components has increased, this property is less relevant and can likely be deprecated in the near future.